src/main/scala/com/lunatech/pointingpoker/Room.scala

```diff
@@ -6,7 +6,7 @@ import akka.actor.{Actor, ActorLogging, ActorRef, Props}
 import com.lunatech.pointingpoker.websocket.WSMessage
 import com.lunatech.pointingpoker.websocket.WSMessage.MessageType
 
-class Room(roomId: UUID) extends Actor with ActorLogging {
+class Room(val roomId: UUID) extends Actor with ActorLogging {
```

Comment:
Total side note,but I've been thinking some about testing lately, especially after seeing @eamelink post about all his testing _learnings_ the other day. While this isn't what he was talking about at all, but I always feel odd when I make a change like this _only_ for testing. Same with taking private methods and making them public just for the sake of being able to test them. It always makes me ask, should I actually do this? Do I need to test this private _unit_, or is testing what is exposed good enough? Just ramblings, ignore them. This would be a cool topic for someone to do a tech talk on. 👀

Reply:
It's a fair point, especially with actors where accessing class members directly is discouraged. In this case I would suggest adding a new command; something along the lines of `GetState` that returns the current users, issue, room ID, etc.

Reply:
I agree that changing the access level only for testing is not so _nice_, however considering that it was inside an actor I don't see as a big problem, since that method shouldn't accessible either way (like any other internal state from the actor). As for the roomId, I didn't see a problem as it is immutable and it is also the actor name, so having as a class value only gives me a more straightforward way of accessing, instead of going via `self.path.name`